File: WorkoutApplication.Domain/Users/UserMinMaxReps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WorkoutApplication.Domain.BaseClasses;

namespace WorkoutApplication.Domain.Users
{
    public class UserMinMaxReps : Entity
    {
        public int UserSettingsId { get; set; }
        public UserGoal Goal { get; set; }
        public SetType SetType { get; set; }
        public ushort? Duration { get; set; }
        public byte MinReps { get; set; }
        public byte MaxReps { get; set; }
    }
}

File: WorkoutTracker.UI/workout-tracker/src/app/core/models/user-min-max-reps.ts
import { SetType } from "../enums/set-type";
import { UserGoal } from "../enums/user-goal";

export class UserMinMaxReps {
  public userSettingsId: number;
  public goal: UserGoal;
  public setType: SetType;
  public duration: number;
  public minReps: number;
  public maxReps: number;
}
